src/Symfony/Component/Form/Extension/Core/Type/PasswordType.php

```diff
@@ -20,7 +20,7 @@ class PasswordType extends AbstractType
 {
     public function buildView(FormView $view, FormInterface $form, array $options): void
     {
-        if ($options['always_empty'] || !$form->isSubmitted()) {
+        if ($options['always_empty'] && !$form->isSubmitted()) {
```

Comment:
This looks wrong to me as it means that form will now be populated with the submitted password even if the `always_empty` option is `true` (which is the default).